lib/interceptor.js

```diff
@@ -263,13 +280,24 @@ var interceptor = {
       );
     }
 
+    function shouldExcludeRequestByUrl(url) {
+      return window[NAMESPACE].excludedUrls.some(function (regex) {
+        return typeof regex.test === 'function'
+          ? regex.test(url)
+          : new RegExp(regex).test(url);
+      });
```

Comment:
Are there any scenarios where the stored item is no longer a regexp? If not, then we can remove the conditional expression